docs/proposals/scheduling/20220629-schedule-device-in-card-level.md

```diff
@@ -0,0 +1,268 @@
+---
+title: schedule device(gpu\rdma) in card level
```

Comment:
This title is confusing. Kubernetes itself supports card granularity allocation. It is recommended to change to Fine-grained GPU Management